addons/mail/static/src/js/owl/component/chat_window.js

```diff
@@ -248,14 +261,18 @@ class ChatWindow extends owl.store.ConnectedComponent {
 }
 
 ChatWindow.defaultProps = {
+    composerInitialAttachmentLocalIds: [],
+    composerInitialTextInputHtmlContent: null,
     dockDirection: 'rtl',
     dockOffset: 0,
     hasCloseAsBackButton: false,
     hasShiftLeft: false,
     hasShiftRight: false,
+    initialState: {},
     isDocked: false,
     isExpandable: false,
     isFullscreen: false,
+    threadInitialScrollTop: null,
```

Comment:
no default props

_(`optional` considers `undefined` like unset, so it passes during props validation. That's not the case for `null`!)_

Reply:
Okay didn't know that !